Move Abakus group verification to after verifying student status

eikhr opened the pull request:
This avoids unverified users getting added to the Abakus-group

Resolves ABA-1168

jonasdeluna reviewed (approved):
Oh lol mb, it seems I added it to the wrong part of the validation.
Great work! :chart_with_upwards_trend: :chart_with_upwards_trend: :chart_with_upwards_trend: :chart_with_upwards_trend:

eikhr commented:
> Great work! Do you think it's necessary to write a test for the lines that weren't covered?

Yes! I have added some tests, and also fixed a bug in my code that caused no one to be added to the Abakus group:)

juniwbjerde reviewed (approved):
Looks good:)
